### PR TITLE
add no stockpile change

### PR DESCRIPTION
**Pull request by CIO61**
disables starting stockpile next to keep, allowing it to be placed anywhere from the get-go

May adversely affect AI's (which arent used often anyways, but still)

### CODE REVIEW

File: Rebalance/config.json [hunk 1]
```diff
@@ -1224,6 +1224,8 @@
 
     {"description": "Projectile Range - cow", "address": "0x1B63B4", "size":4, "value": 60},
     {"description": "Manual Control Range - Catapult, Cow Throw", "address": "0x3597C", "size":4, "value": 3600},
-    {"description": "Manual Control Range 2 - Catapult, Cow Throw", "address": "0x36358", "size":4, "value": 3600}
+    {"description": "Manual Control Range 2 - Catapult, Cow Throw", "address": "0x36358", "size":4, "value": 3600},
+  
+    {"description": "Disable starting stockpile. EB 0F [jump over 15 bytes]", "address": "0x115136", "size": 1, "value": [235, 15]}
```
Comment (Krarilotus) [inline Rebalance/config.json]: **AI starting stockpiles are removed too.** On the checked Extreme 1.41 executable, `EB 0F` jumps from `0x00515136` to `0x00515147`, skipping the stockpile-placement call at `0x00515142` to `0x005088C0`. This is the shared keep-placement tail and has no human/AI filter. Native execution of this block confirms that the original calls for every owner slot 1–8, while the patch skips all of them.

All 16 bundled Ascension AIVs contain explicit stockpile steps, so this does not prove every AI permanently stalls. However, some first steps are late (Marshal 39, Snake 52), and recovery depends on build progress, resources and usable map placement. The requirement that AI starting stockpiles remain correctly placed is therefore not met.

Comment (Krarilotus) [inline Rebalance/config.json]: **This offset must not be copied unchanged into a UCP3 port.** It matches the Extreme executable used by the legacy setup, but the same file offset in the checked standard Crusader 1.41 executable is inside `placeWheatfarm` and would overwrite part of an instruction. The corresponding standard keep-stockpile block is `0x00514DB6`–`0x00514DC7`, calling `placeStockpile` at `0x00508540`.

For clarity, `size: 1` with `[235, 15]` is valid for the legacy loader: it writes one byte per list element at consecutive addresses. The issue is executable-specific addressing and missing AI filtering, not a truncated patch.

**Review by Krarilotus — CHANGES_REQUESTED** [review]
**TL;DR:** We are leaving this PR unmerged because it also removes AI starting stockpiles, and it is a legacy change rather than an implementation for the current UCP3 Ascension branch.

The multiplayer owner selection looks correct: the native stockpile validator uses the supplied player's stockpile entry, not the local player's. All 256 isolated assembly checks passed (8 owner slots × 8 local-player slots × 4 stockpile states), with count/adjacency callees stubbed and their owner arguments verified. This is not a live multiplayer or AI economy test, so it does not establish full-match safety.

The current PR targets `main` and modifies `Rebalance/config.json`, which is absent from `ucp3-ascension`. Any future version of this feature would need an actual UCP3 implementation on `ucp3-ascension`, explicit preservation of AI starting placement, and gameplay validation. Version bumps and an extension-store export through branch `3.0.7` should wait until such a change is accepted. No merge or release is planned for this PR.

A separate, behavior-preserving stockpile-cleanup reimplementation discovered during this investigation is filed as [OpenSHC #217](https://github.com/sourcehold/OpenSHC/pull/217); it does not implement starting-stockpile removal.
